testutils/assert/assert.go

```diff
@@ -0,0 +1,57 @@
+package assert
```

Comment:
I wouldn't want to risk illness...

Yes, the vendoring situation is a mess! Also bugs me that the golang authors didn't include these utterly basic/obvious `assert`-like testing utilities that absolutely everyone wants to use. Oh well.